mod/05_email/set/right/bcc.rb

```diff
@@ -18,7 +22,7 @@ def process_email_addresses context_card, format_args, args
 format :email_text do
   view :email_addresses do |args|
     context = args[:context] || self
-    card.item_names.map do |item_name|
+    card.item_names(:context=>context.cardname).map do |item_name|
```

Comment:
Hmm, what does the :context arg do here?  It might make sense that it allows item_names to absolutize the names to the context, but doesn't that happen in the next line?